fulp_modules/jobs/deputy/code/deputy.dm

```diff
@@ -0,0 +1,115 @@
+/datum/job/fulp/deputy
+	title = "Deputy"
+	auto_deadmin_role_flags = DEADMIN_POSITION_SECURITY
+	department_head = list("Head of Security")
+	faction = "Station"
+	total_positions = 4
+	spawn_positions = 4
+	supervisors = "the head of your assigned department, and the head of security when outside your post"
+	selection_color = "#ffeeee"
+	minimal_player_age = 7
+	exp_requirements = 300
+	exp_type = EXP_TYPE_CREW
+	exp_type_department = EXP_TYPE_SECURITY
+	id_icon = 'fulp_modules/jobs/cards.dmi'
+	hud_icon = 'fulp_modules/jobs/huds.dmi'
+	fulp_spawn = /obj/effect/landmark/start/deputy
+
+	access = list(ACCESS_SECURITY, ACCESS_BRIG, ACCESS_SEC_DOORS, ACCESS_MAINT_TUNNELS)
+	minimal_access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS)
```

Comment:
without weapon access beepsky will chase them for having a stun baton

Reply:
Only the one in the armory should, not sure about the one in AI sat, but deputies have no reason to be there anyways

Reply:
sure